<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkspaceComposer</class>
 <widget class="QMainWindow" name="WorkspaceComposer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1074</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>30</y>
      <width>281</width>
      <height>321</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEditGroupName">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>NewWorkspaceGroup</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Group Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Group Comment</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditGroupComment">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>171</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>NewGroupComment</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonSelectAll">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>100</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Select All</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonClearAll">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>120</y>
       <width>82</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear All</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonRemoveSelected">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>140</y>
       <width>111</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove Selected</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonChooseWorkspace">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>211</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Choose Workspace(s)</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButtonUpdate">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>110</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Update</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxEnforceCompatibility">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>200</y>
       <width>231</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Enforce Workspace Compatibility</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxSumWorkspaces">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>180</y>
       <width>181</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Sum Workspaces</string>
     </property>
    </widget>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>370</y>
      <width>751</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Workspace Algebra Section</string>
    </property>
    <widget class="QLineEdit" name="lineEditResultingWorkspace">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Enter the name of the workspace to contain the results of the equation</string>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
border-color: rgb(0, 0, 0);</string>
     </property>
     <property name="text">
      <string>AlgWorkspace</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>50</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>=</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEditEquation">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>50</y>
       <width>471</width>
       <height>20</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Enter an equation using the workspace indicies listed in the Workspace List above.  The shorter Index names are used for convenience.</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>191</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Resulting Workspace Name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>30</y>
       <width>471</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Enter Equation - use workspace indicies, for example:  (ws1-ws2)/ws3</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>370</y>
      <width>281</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Perform Task</string>
    </property>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>156</width>
       <height>31</height>
      </rect>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelStatusMsg">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Status Message:</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelNumWorkspaces">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>261</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Number of Workspaces: </string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>751</width>
      <height>341</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);
</string>
    </property>
    <property name="title">
     <string>Workspace List</string>
    </property>
    <widget class="QTableWidget" name="tableWidgetWGE">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>730</width>
       <height>300</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>200</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>730</width>
       <height>300</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>5</width>
       <height>5</height>
      </size>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="rowCount">
      <number>15</number>
     </property>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>18</number>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>15</number>
     </attribute>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <row/>
     <column>
      <property name="text">
       <string>Index</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Workspace</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Location</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Date</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Size</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Select</string>
      </property>
     </column>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1074</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>buttonBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>867</x>
     <y>312</y>
    </hint>
    <hint type="destinationlabel">
     <x>867</x>
     <y>312</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>centralwidget</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>centralwidget</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>534</x>
     <y>206</y>
    </hint>
    <hint type="destinationlabel">
     <x>534</x>
     <y>206</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>buttonBox</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>967</x>
     <y>336</y>
    </hint>
    <hint type="destinationlabel">
     <x>967</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButtonUpdate</sender>
   <signal>clicked(bool)</signal>
   <receiver>pushButtonUpdate</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>987</x>
     <y>192</y>
    </hint>
    <hint type="destinationlabel">
     <x>987</x>
     <y>192</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
